src/sourmash/command_compute.py

```diff
@@ -188,12 +191,13 @@ def _compute_individual(args, signatures_factory):
                 add_seq(sigs, record.sequence,
                         args.input_is_protein, args.check_sequence)
 
-            notify('...{} {} sequences', filename, n, end='')
+            if n is not None:       # don't write out signatures if no input
```

Comment:
is it worth notifying that the input was empty?

Reply:
yes, thank you! updated.